modules/swagger-codegen/src/main/resources/qt5cpp/helpers-header.mustache

```diff
@@ -21,6 +22,19 @@ namespace {{this}} {
     QString stringValue(qint64 value);
     QString stringValue(bool value);
 
+    template <typename ObjPtrT>
```

Comment:
maybe move this class to its own file

Reply:
Do you want to suggest a more appropriate class name also?

Reply:
OK, I came up with object wrapper which is what it is..., :)